src/components/Sorting/Sorting.jsx

```diff
@@ -0,0 +1,77 @@
+import { useEffect, useState } from 'react';
+import addSort from 'src/img/plus.png';
+import { fields } from 'src/constants';
+import './style.scss';
+
+const Sorting = ({list, setList}) => { 
+    
+  const [sortingField, setSortingField] = useState('');
+  const [openSortDirection, setOpenSortDirection] = useState(false); 
+  const [sortingWay, setSortingWay] = useState('asc');
+
+  const sort = () => {    
+    const sortedList = [...list].sort((a, b) => (    
+      (a[sortingField] > b[sortingField]) ? -1 : ((a[sortingField] < b[sortingField]) ? 1 : 0))  
+    );
+    if (sortingWay === 'desc') sortedList.reverse();         
+    setList(sortedList);    
+  } 
+
+  useEffect( () => {         
+    sort();     
+  }, [sortingField, sortingWay]); 
+
+  return (
+    <div className='sorting'>    
+      <div className='sorting-body'>
+        <div className='sorting-block'>
+          <label className='sorting-block_label__field' htmlFor='sorting-block_select__field'>Сортировать по:</label>
+          <select 
+            className='sorting-block_select__field'
+            onClick={() => setOpenSortDirection(true)}
+            onChange={(e) => setSortingField(e.target.value)}
+          >
+            <option></option>          
+            {
+              fields.map((item, index) => (
+                <option 
+                  value={item.value} 
+                  key={index}
+                  className='sorting-block_option__field'               
+                >
+                  {item.name}
+                </option>
+              ))        
+            }
+          </select>
+        </div>        
+        <div className={openSortDirection ? 'sorting-block__direction active' : 'sorting-block__direction '} >

```

Comment:
it reminds me of something) 
`sorting-block__direction`
Again, you're just hiding the element, not deleting it.